src/cil.ml

```diff
@@ -2207,6 +2226,7 @@ let rec alignOf_int t =
     | TInt((IInt|IUInt), _) -> !M.theMachine.M.alignof_int
     | TInt((ILong|IULong), _) -> !M.theMachine.M.alignof_long
     | TInt((ILongLong|IULongLong), _) -> !M.theMachine.M.alignof_longlong
+    | TInt((IInt128|IUInt128), _) -> 16 (* ? *)
```

Comment:
Any reason why we do not generate this via the  machine settings?

Reply:
Can this ever be something else?
Generating it may not work on architectures don't support 128bit ints.
https://en.m.wikipedia.org/wiki/128-bit_computing#Software